doc/user/content/sql/create-sink.md

```diff
@@ -163,6 +165,22 @@ You can find the topic name for each Kafka sink by querying `mz_kafka_sinks`.
 
 {{% kafka-sink-drop  %}}
 
+#### Enabling topic reuse after restart
+
+{{< version-added v0.8.2 />}}
+
+This feature is only available in experimental mode. By default, Materialize creates new, distinct topics for sinks after each restart. To enable the reuse of the existing topic instead, when you create a sink, you must:
+
+* Enable the `exactly_once` switch.
+* Specify a consistency topic to store the information Materialize will use to identify where synching left off. For more information, see [Consistency metadata](#consistency-metadata).
```

Comment:
I think "synching" in this sentence refers to the writes made by the sink? IMO it would be clearer to avoid any words like "sync" when talking about "sinks" and the sentence would be clearer as 
"Specify a consistency topic to store the information Materialize will use to identify the last completed write."